Fixes a cross compilation error on aarch64 (possible others too):

  /builddir/firefox-62.0.3/media/webrtc/trunk/webrtc/common_audio/signal_processing/complex_bit_reverse.c:20:22: error: redefinition of 'index_7'

--- media/webrtc/trunk/webrtc/common_audio/signal_processing/complex_bit_reverse.c.orig	2018-10-27 21:12:37.895516483 +0200
+++ media/webrtc/trunk/webrtc/common_audio/signal_processing/complex_bit_reverse.c	2018-10-27 21:13:49.208516829 +0200
@@ -16,6 +16,8 @@
  * operation. Same for index_8.
  */
 
+#ifndef __COMPLEX_BIT_REVERSE_C
+#define __COMPLEX_BIT_REVERSE_C
 /* Indexes for the case of stages == 7. */
 static const int16_t index_7[112] = {
   1, 64, 2, 32, 3, 96, 4, 16, 5, 80, 6, 48, 7, 112, 9, 72, 10, 40, 11, 104,
@@ -106,3 +108,4 @@ void WebRtcSpl_ComplexBitReverse(int16_t
     }
   }
 }
+#endif /* __COMPLEX_BIT_REVERSE_C */
--- media/webrtc/trunk/webrtc/common_audio/signal_processing/filter_ar_fast_q12.c.orig	2018-10-27 22:51:02.032543685 +0200
+++ media/webrtc/trunk/webrtc/common_audio/signal_processing/filter_ar_fast_q12.c	2018-10-27 22:50:50.567543631 +0200
@@ -8,6 +8,8 @@
  *  be found in the AUTHORS file in the root of the source tree.
  */
 
+#ifndef __FILTER_AR_FAST_Q12
+#define __FILTER_AR_FAST_Q12
 #include "webrtc/base/checks.h"
 #include "webrtc/common_audio/signal_processing/include/signal_processing_library.h"
 
@@ -40,3 +42,4 @@ void webrtcspl_filterarfastq12(const int
     data_out[i] = (int16_t)((output + 2048) >> 12);
   }
 }
+#endif /* __FILTER_AR_FAST_Q12 */
--- ./media/webrtc/trunk/webrtc/common_audio/signal_processing/spl_sqrt_floor.c.orig	2018-10-27 23:03:30.592547144 +0200
+++ ./media/webrtc/trunk/webrtc/common_audio/signal_processing/spl_sqrt_floor.c	2018-10-27 23:04:29.741547419 +0200
@@ -28,6 +28,8 @@
 
 // Minor modifications in code style for WebRTC, 2012.
 
+#ifndef __SPL_SQRT_FLOOR
+#define __SPL_SQRT_FLOOR
 #include "webrtc/common_audio/signal_processing/include/signal_processing_library.h"
 
 /*
@@ -75,3 +77,4 @@ int32_t WebRtcSpl_SqrtFloor(int32_t valu
 
   return root >> 1;
 }
+#endif /* __SPL_SQRT_FLOOR */
--- ./media/webrtc/trunk/webrtc/modules/audio_coding/codecs/isac/fix/source/lattice_c.c.orig	2018-10-27 23:20:58.921551910 +0200
+++ ./media/webrtc/trunk/webrtc/modules/audio_coding/codecs/isac/fix/source/lattice_c.c	2018-10-27 23:21:34.043552068 +0200
@@ -14,6 +14,8 @@
  *
  */
 
+#ifndef __LATTICE_C
+#define __LATTICE_C
 #include "settings.h"
 #include "signal_processing_library.h"
 #include "webrtc/typedefs.h"
@@ -47,3 +49,4 @@ void WebRtcIsacfix_FilterArLoop(int16_t*
     ar_g_Q0[0] = tmpAR;
   }
 }
+#endif /* __LATTICE_C */
